samcli/commands/local/lib/swagger/parser.py

```diff
@@ -263,7 +263,9 @@ def get_default_authorizer(self, event_type: str) -> Union[str, None]:
 
         if not document_version.startswith(SwaggerParser._3_X_VERSION) or not event_type == Route.HTTP:
             raise IncorrectOasWithDefaultAuthorizerException(
-                "Root level definition of default authorizers are only supported for OpenApi 3.0"
+                "Root level definition of default authorizers are only supported for "
+                "AWS::Serverless::HttpApi and AWS::ApiGatewayV2::Api resources using "
```

Comment:
🤔 hmm, we could provide a link to the docs, however the docs basically say the same thing that is being said in the message: root level default authorizers aren't support because API Gateway V1 doesn't use root level default authorizers